index.html

```diff
@@ -1,112 +1,135 @@
-						<img src="assets/icons/services/cooking.svg" alt="Cooking">
-					</div>
-            <p class="service__description" >Restaurante de cocina latinoamericana</p>
+    </section>
+    <section class="homepage-hero">
+      <!-- Hero section, the cover for the page -->
+      <div class="homepage-hero__layer"></div>
+      <article class="homepage-hero__article">
+        <div class="homepage-hero__headline">
+          <h1>Hostal</h1>
+          <h1>Las paisas</h1>
+        </div>
+        <div class="homepage-hero__divider"></div>
+        <span class="homepage-hero__promotext">El paraiso te espera</span>
+      </article>
+    </section>
+    <section class="carrousel-section">
+      <div class="container">
+        <div class="info">
+          <h1 class="info-title">Comodidades</h1>
+          <hr class="info-line">
+          <p class="info-description">
+          Nuestro hostal te ofrece todas las comodidades para que vivas la mejor experiencia.
+          </p>
+        </div>
+        <div id="carrouselContainer" class="carrousel-container">
+          <div class="carrousel">
+            <img src="assets/images/IMG_8056.JPG" class="carrousel__img"/>
+            <img src="assets/icons/arrow.svg" class="carrousel__arrow carrousel__arrow--left" data-direction="back"/>
+            <img src="assets/icons/arrow.svg" class="carrousel__arrow carrousel__arrow--right" data-direction="ahead"/>
           </div>
-       <div class="service">
-          <div class="service__icon">
-						<img src="assets/icons/services/reception.svg" alt="Reception">
-					</div>
-            <p class="service__description" >Recepción las 24 horas</p>
-       </div>
+        </div>
+          <div class="pointer-section">
+            <div class="pointer-list" id="divBtn">
+              <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn1" ></div>
+              <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn2" ></div>
+              <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn3" ></div>
+              <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn4" ></div>
+            </div>
+          <button href="#" class="button button--medium button--secondary">Ver galeria</button>
+        </div>
+      </div>
+    </section>
+    <section class="services-section">
+      <div class="container">
+        <div class="services-container">
+          <div class="service">
+            <div class="service__icon">
+              <img src="assets/icons/services/sunset.svg" alt="Sunset">
+            </div>
+              <p class="service__description">Zona de playa privada</p>
+            </div>
+          <div class="service">
+            <div class="service__icon">
+              <img src="assets/icons/services/cooking.svg" alt="Cooking">
+            </div>
+              <p class="service__description" >Restaurante de cocina latinoamericana</p>
+            </div>
```

Comment:
please correct the indentation

Reply:
These were issues coming from a PR merged by @ArleyAV , we have to be more careful with this kind of things. I fixed it.